examples/django/oauth_app/slack_datastores.py

```diff
@@ -64,10 +102,12 @@ def find_bot(
                 enterprise_id=b.enterprise_id,
                 team_id=b.team_id,
                 bot_token=b.bot_token,
+                bot_refresh_token=b.bot_refresh_token,
+                bot_token_expires_at=b.bot_token_expires_at,
                 bot_id=b.bot_id,
                 bot_user_id=b.bot_user_id,
                 bot_scopes=b.bot_scopes,
-                installed_at=b.installed_at.timestamp(),
```

Comment:
This conversion is no longer necessary as `Bot` / `Installation` internally do the same for you